pom.xml

```diff
@@ -67,6 +67,9 @@
             <id>jdk21_profile</id>
             <properties>
                 <jdk.version>21</jdk.version>
+                <spotlessExcludedFiles>
+                    **/SwitchBenchmark.java
```

Comment:
Spotless does not yet handle `when` in switch expressions and will throw an error. I am deactivating it for this file for now.

Reply:
Good, but I guess you need to add it also to the jdk11 and jdk17 profiles as well